x-pack/plugins/cloud_security_posture/public/components/cloud_posture_page.tsx

```diff
@@ -275,7 +275,12 @@ export const CloudPosturePage = <TData, TError>({
       return defaultLoadingRenderer();
     }
 
-    if (getSetupStatus.data.status === 'not-installed') {
+    /* Checks if its a completely new user which means no integration has been installed and no latest findings default index has been found */
+    if (
+      getSetupStatus.data?.kspm?.status === 'not-installed' &&
+      getSetupStatus.data?.cspm?.status === 'not-installed' &&
+      getSetupStatus.data?.indicesDetails[0].status === 'empty'
```

Comment:
which index are you checking for here? Is the `'logs-cloud_security_posture.findings_latest-default'` is always the first index.

Reply:
yes, its always the first one from my understanding
I should probably add a comment there to make things clear